src/applications/disability-benefits/all-claims/tests/fixtures/data/maximal-bdd-test.json

```diff
@@ -224,7 +224,10 @@
           }
         ],
         "view:militaryHistoryNote": {},
-        "separationLocation": "Andersen AFB - Air Force / Guam"
+        "separationLocation": {
+          "id": "123",
+          "label": "Test"
```

Comment:
We can use this test data in the save in progress menu, so it'd be nice if this had a "real" content. I guess we could make the ID match values in staging since that is where it would most likely get used.

Reply:
That's a great idea. I will change it to use one of the previous staging values.